inc/languages/english/myalerts.lang.php

```diff
@@ -16,7 +16,7 @@
 
 $l['myalerts_modal_title'] = 'Recent Alerts';
 $l['myalerts_modal_display_alerts'] = 'View Alerts';
-$l['myalerts_modal_unread_only'] = 'Unread Only';
+$l['myalerts_modal_unread_only'] = 'Display Unread Only';
```

Comment:
Is this really unclear enough that it needs that extra word? If so, we need to also increase the `width` style of `#myalerts_alerts_modal` from 450px to about 470px, otherwise the footer breaks into two lines on the default theme.

Reply:
Could be confusing... but I ma not a native speaker .) so tell me :)

Reply:
A proposal:

You update this PR so as to change "Display" to "Show" (which is slightly shorter), and to change the `width` style of `#myalerts_alerts_modal` from `450px` to `460px`.

So long as it passes a syntax check, I then merge this PR without further discussion.

Yes?